src/components/card-lite/CardLite.tsx

```diff
@@ -0,0 +1,31 @@
+import "./CardLite.css";
+import { GoAButton } from "@abgov/react-components";
+import { ReactNode } from "react";
+import { useNavigate } from "react-router-dom";
+
+export interface Props {
+  title: string;
+  description: string;
+  linkTo: string;
+  linkDisplay: ReactNode;
+}
+
+export function CardLite(props: Props) {
+  const navigate = useNavigate();
+  return (
+    <div className="card-lite">
+      <div className="title">{props.title}</div>
+      <div className="description">{props.description}</div>
+      <GoAButton
+        type="tertiary"
+        onClick={() => {
+          navigate(props.linkTo);
+        }}
+      >
+        {props.linkDisplay}
+      </GoAButton>
+    </div>
```

Comment:
I talked to @twjeffery about replacing these GoAButtons with links and he was in agreement. Using buttons for navigation is not ideal for a variety of reasons and we shouldn't deviate from the standard. It will also remove the unwanted padding of the link text on the screen, bring the links inline with the text above.